packages/dashboard/src/components/ConfirmDialog/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { MaskDialog } from '@dimensiondev/maskbook-theme'
+import {
+    DialogContent,
+    DialogActions,
+    Button,
+    dialogActionsClasses,
+    dialogContentClasses,
+    experimentalStyled as styled,
+    buttonClasses,
+} from '@material-ui/core'
+
+const StyledDialogActions = styled(DialogActions)(({ theme }) => ({
+    [`&.${dialogActionsClasses.root}>:not(:first-of-type)`]: {
+        marginLeft: theme.spacing(3),
```

Comment:
It seems like you've already add this to the theme. Do you still need it?

Reply:
Will remove this one